frontend/src/lib/api/urls.ts

```diff
@@ -8,6 +8,15 @@ export function userUrl(user: User): string {
     return `/users/${user.username}`
 }
 
+export function userAvatarUrl(user: User): string | null {
+    return user.github_id && `https://avatars.githubusercontent.com/u/${user.github_id}`
+}
+
+// Only returns a URL if the user has a GitHub ID
+export function userGithubHtmlUrl(user: User): string | null {
+    return user.github_id && `https://github.com/${user.username}`
```

Comment:
Better to use `??` over `&&` I think

Reply:
that makes it return the id if it's present apparently, which isn't the behavior I want here